src/webserver/Server.h

```diff
@@ -130,44 +144,39 @@ class Server : public std::enable_shared_from_this<Server<PlainSession, SslSessi
         tcp::endpoint endpoint,
         util::TagDecoratorFactory tagFactory,
         clio::DOSGuard& dosGuard,
-        std::shared_ptr<Callback> const& callback)
+        std::shared_ptr<Handler> const& callback)
         : ioc_(std::ref(ioc))
         , ctx_(ctx)
         , tagFactory_(std::move(tagFactory))
         , dosGuard_(std::ref(dosGuard))
-        , callback_(callback)
+        , handler_(callback)
         , acceptor_(boost::asio::make_strand(ioc))
     {
         boost::beast::error_code ec;
 
-        // Open the acceptor
         acceptor_.open(endpoint.protocol(), ec);
         if (ec)
             return;
 
-        // Allow address reuse
         acceptor_.set_option(boost::asio::socket_base::reuse_address(true), ec);
         if (ec)
             return;
 
-        // Bind to the server address
         acceptor_.bind(endpoint, ec);
         if (ec)
         {
             log_.error() << "Failed to bind to endpoint: " << endpoint << ". message: " << ec.message();
-            throw std::runtime_error("Failed to bind to specified endpoint");
+            throw std::runtime_error(fmt::format("Failed to bind to endpoint:{}", endpoint.address().to_string()));
```

Comment:
I hope it includes the port ❓

Reply:
If it's not included and you will be changing this to add the port, please also add a space after `:`.
`... at endpoint: 127.0.0.1:51233` is much nicer than `... at endpoint:127.0.0.1:51233`.